ResidentsSocialCarePlatformApi/V1/Domain/PersonalRelationships.cs

```diff
@@ -4,6 +4,14 @@ namespace ResidentsSocialCarePlatformApi.V1.Domain
 {
     public class PersonalRelationships
     {
+        public PersonalRelationships()
+        {
+            Parents = new List<long>();
+            Siblings = new List<long>();
+            Children = new List<long>();
+            Other = new List<long>();
+        }
+
         public List<long> Parents { get; set; }
```

Comment:
`public List<long> Parents { get; set; } = new List<long>(); ` syntax could be used here as well. Using constructor like that gives the same end result of course.

Reply:
I actually wanted to keep them separate, as I like to initialise them in the constructor.
Let me know if you prefer the other syntax, I can change it :)

Reply:
That's absolutely fine. You can use whichever syntax you prefer